go-controller/pkg/util/node_annotations.go

```diff
@@ -1072,7 +1072,7 @@ func GetNodeHostAddrs(node *kapi.Node) ([]string, error) {
 	if err != nil && !IsAnnotationNotSetError(err) {
 		return nil, fmt.Errorf("failed to get node host CIDRs for %s: %s", node.Name, err.Error())
 	}
-	return hostAddresses.UnsortedList(), nil
+	return sets.List(hostAddresses), nil
```

Comment:
I think it makes total sense here to sort it so that we have a consistency in deepequal (I have faced similar issues in other parts of the code where I had to do this)